win32/help/file.d

```diff
@@ -14,23 +12,18 @@ the ability to do exclusive or shared locking. However, what it does
 not give you is the ability to switch between blocking/non-blocking
 (it fails immediately -- does not block) To do that, you need to use
 LockfileEx -- which can even lock a specific part of a file.
-
 <nl>The basic procedure for doing this is to first call Createfile to 
 give you a filehandle. Then call LockfileEx with the filehandle.
 Do whatever to the file. Call UnlockfileEx. Then close the filehandle.
 (Some of you may want to close the filehandle to kill the locks, it
 doesn't work that way with win32, at least according to the msdn)
-
 Below is a class called Flock, which gives you exclusive/shared
 locking with non-blocking/blocking abilities. If you can think of any
 optimizations or changes, be sure to let me know. 
-
```

Comment:
Have you checked how this now renders? I'm worried that removing many of these blank lines will hurt readability - especially between human text and examples. What is your motivation for touching this file at all?